fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -42,19 +42,37 @@ typedef Reference<IDataDistributionTeam> ITeamRef;
 typedef std::pair<ITeamRef, ITeamRef> SrcDestTeamPair;
 
 // TODO: add guard to guarantee the priority is not equal for each purpose?
+// Fix it by adding enum variables to reprensent PRIORITY.
 inline bool isDiskRebalancePriority(int priority) {
```

Comment:
Isn't the goal to replace old way of setting rebalance priority with new one? Do we still need old functions?